config.minimal.yml

```diff
@@ -14,3 +14,157 @@ session:
   # The domain to protect.
   # Note: Authelia must also be served by that domain.
   domain: example.com
+
+# Configuration of the storage backend used to store data and secrets.
```

Comment:
I would reduce the amount of comments in this configuration because it was made to be simple to read for users and with the least number of options to not make people confused.
I suggest we only leave the most useful comments and not list all possibilities for an option in the comments.

Reply:
I kept them in, since I thought youd want it to be like the other config. But that was just my assumption. Ill look at it and see which I think should definitely stay and you can then tell me if you agree.